gradle-plugin/src/main/kotlin/co/touchlab/kotlin/gradle/targets/native/cocoapods/CocoapodsExtension.kt

```diff
@@ -47,17 +48,20 @@ open class CocoapodsExtension(private val project: Project) {
     @Input
     var homepage: String? = null
 
-    /**
-     * Configure framework name of the pod built from this project.
-     */
-    @Input
-    var frameworkName: String = project.name.asValidFrameworkName()
+    internal var frameworkConfiguration: Framework.() -> Unit = {
+        baseName = project.name.asValidFrameworkName()
+        isStatic = true
+    }
```

Comment:
We're defaulting `baseName` and `isStatic`. If we add a block to config a framework value
```kotlin
framework {
  someOtherThing = "abc"
}
```
That will make `isStatic` false and possibly change `baseName` because the defaults are no longer set.

Reply:
updated to set defaults first on every framework